bears/c_languages/CPPCleanBear.py

```diff
@@ -22,5 +23,12 @@ class CPPCleanBear:
     CAN_DETECT = {'Smell', 'Unused Code', 'Security'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return filename,
+    def create_arguments(filename,
+                         file,
+                         config_file,
+                         include_paths:
```

Comment:
please put the type annotation onto the same line as `include_paths`, that reads way better